hws/src/dr_common_functions.py

```diff
@@ -164,11 +179,20 @@ def dr_parse_remove_header2header_action(action_dw_0, action_dw_1):
 
 def dr_parse_insert_inline_action(action_dw_0, action_dw_1, parse_value=True):
     action = {"type" : "Insert inline"}
-    start_anchor = int(action_dw_0[10 : 16], 2)
+    start_anchor = 0
+    start_offset = 0
+
+    if _config_args.get("cx8"):
+        start_anchor = int(action_dw_0[8 : 15], 2)
+        start_offset = int(action_dw_0[15 : 23], 2)
+    else:
+        start_anchor = int(action_dw_0[10 : 16], 2)
+        start_offset = int(action_dw_0[16 : 23], 2)
```

Comment:
The original code says action_dw_0[18 : 23], not [16 : 23].
Which one is the correct one?

Reply:
it should be like the new one